src/main/groovy/life/qbic/utils/NanoporeParser.groovy

```diff
@@ -74,9 +77,10 @@ class NanoporeParser {
         }
         def finalMetaData = (Map) jsonSlurper.parseText(buffer.toString())
 
-        new File(summaryFile["path"].toString()).readLines().each { line ->
-            def split = line.split("=")
-            finalMetaData[split[0]] = split[1]
+        new File(root.toString() + summaryFile["path"].toString())
```

Comment:
Dangerous Path joining. If the root does not end with `/` xor the summary file does not start with `/` and if the root ends with and the summary file starts with `/` this is not handled properly.
```suggestion
        new File(Paths.get(root.toString(), summaryFile["path"].toString()))
```